frontend/packages/core/src/stories/timeseries.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { Meta } from "@storybook/react";
+
+import TimeseriesChart, { TimeseriesReferenceLineProps } from "../Charts/timeseries";
+
+export default {
+  title: "Core/TimeseriesChart",
+  component: TimeseriesChart,
+} as Meta;
+
+const ChartContainer = styled("div")({
+  width: 900,
+  height: 400,
+});
+
+export const SingleDataLine = () => {
+  const mockDataSingleLine = [
+    {
+      lineName: "line1",
+      timestamp: 1546300800000,
+      value: 5,
+    },
+    {
+      lineName: "line1",
+      timestamp: 1546300900000,
+      value: 20,
+    },
+    {
+      lineName: "line1",
+      timestamp: 1546301000000,
+      value: 30,
+    },
+    {
+      lineName: "line1",
+      timestamp: 1546300700000,
+      value: 5,
+    },
+    {
+      lineName: "line1",
+      timestamp: 1546300600000,
+      value: 20,
+    },
+    {
+      lineName: "line1",
+      timestamp: 1546301800000,
+      value: 30,
+    },
+  ];
+
+  const mockLines = [
+    {
+      dataKey: "value",
+      color: "red",
+      animationDuration: 0,
+    },
+  ];
+
+  return (
+    <ChartContainer>
+      <TimeseriesChart
+        data={mockDataSingleLine}
+        xAxisDataKey="timestamp"
+        yAxisDataKey="value"
+        lines={mockLines}
+        xDomainSpread={0.3}
+        yDomainSpread={0.3}
+        regularIntervalTicks
+      />
+    </ChartContainer>
+  );
+};
+
+export const MultiLine = () => {
+  const mockDataMultiLine = [
+    {
+      lineName: "line1",
+      timestamp: 1546201800,
+      value2: 15,
+    },
+    {
+      lineName: "line1",
+      timestamp: 1546291900,
+      value2: 20,
+    },
+    {
+      lineName: "line1",
+      timestamp: 1546302000,
+      value2: 80,
+    },
+    {
+      lineName: "line2",
+      timestamp: 1546201820,
+      value: 10,
+    },
+    {
+      lineName: "line2",
+      timestamp: 1546291930,
+      value: 20,
+    },
+    {
+      lineName: "line2",
+      timestamp: 1546302040,
+      value: 40,
+    },
+  ];
+
+  const mockMultipleLines = [
+    {
+      dataKey: "value",
+      color: "red",
+      animationDuration: 0,
+    },
+    {
+      dataKey: "value2",
+      color: "blue",
+    },
+  ];
+
+  return (
+    <ChartContainer>
+      <TimeseriesChart
+        data={mockDataMultiLine}
+        xAxisDataKey="timestamp"
+        yAxisDataKey="value2"
+        lines={mockMultipleLines}
+        singleLineMode={false}
+        xDomainSpread={0.3}
+        yDomainSpread={0.3}
+      />
+    </ChartContainer>
+  );
+};
+
+export const ReferenceLinesAndThingsDisabled = () => {
+  const mockData = [
+    {
+      lineName: "line1",
+      timestamp: 1546301000000,
```

Comment:
i noticed that these millis since epoch just show up as raw numbers in the story. intentional?

Reply:
Yes that is intentional, the purpose of that story is to show people that they don't have to be forced to use a specific time format, and can allow for the raw values as well.

Reply:
added a comment to the story to help explain